src/components/VIcon/VIcon.js

```diff
@@ -44,7 +43,10 @@ export default {
       case props.xLarge: fontSize = SIZE_MAP.xLarge
         break
     }
-    data.style = { fontSize, ...data.style }
+
+    if (fontSize !== undefined) {
```

Comment:
I think it's safe to assume `if (fontSize)`

Reply:
I've updated my PR with this change.